File: AndroidApp/MultiSportTrainer-Mobile/app/src/main/java/com/example/multisporttrainer/models/StartTrainingRequest.java
package com.example.multisporttrainer.models;

public class StartTrainingRequest {

    private int userId;
    private String routeType;
    private String difficulty;
    private String trainingType;
    private int conesCount;
    private int rounds;
    private boolean distractionsEnabled;

    public StartTrainingRequest(
            int userId,
            String routeType,
            String difficulty,
            String trainingType,
            int conesCount,
            int rounds,
            boolean distractionsEnabled
    ) {
        this.userId = userId;
        this.routeType = routeType;
        this.difficulty = difficulty;
        this.trainingType = trainingType;
        this.conesCount = conesCount;
        this.rounds = rounds;
        this.distractionsEnabled = distractionsEnabled;
    }
}

File: Backend/MultiSportTrainerAPI/MultiSportTrainerAPI/DTOs/StartTrainingRequest.cs
namespace MultiSportTrainerAPI.DTOs
{
    public class StartTrainingRequest
    {
        public int UserId { get; set; }

        public string RouteType { get; set; } = string.Empty;

        public string Difficulty { get; set; } = string.Empty;

        public string TrainingType { get; set; } = string.Empty;

        public int ConesCount { get; set; }

        public int Rounds { get; set; }

        public bool DistractionsEnabled { get; set; }
    }
}